### PR TITLE
lightning: add description about terminator for strict-format CSV files

### PR DESCRIPTION
**Pull request by lance6716**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb/pull/53444

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by lance6716** [comment]
more LTS versions will be decided when triage

File: migrate-from-csv-files-to-tidb.md [hunk 1]
```diff
@@ -102,6 +104,8 @@ pd-addr = "${ip}:${port}"     # 集群 PD 的地址，Lightning 通过 PD 获取
 - delimiter 为空；
 - 每个字段不包含 CR (\\r）或 LF（\\n）。
 
+严格格式的 CSV 文件需要显式指定行结束符 `terminator`。
```
Comment (hfxsd) [inline migrate-from-csv-files-to-tidb.md:107]:
```suggestion
严格格式 `strict-format` 的 CSV 文件需要显式指定行结束符 `terminator`。
```

File: migrate-from-csv-files-to-tidb.md [hunk 2]
```diff
@@ -61,6 +61,8 @@ data-source-dir = "${data-path}" # 本地或 S3 路径，例如：'s3://my-bucke
 separator = ','
 # 引用定界符，可以为零或多个字符。
 delimiter = '"'
+# 行结束符。默认同时将 \r, \n, \r\n 作为行结束符
```
Comment (qiancai) [inline migrate-from-csv-files-to-tidb.md:64]:
```suggestion
# 行结束符。默认将 \r、 \n、\r\n 作为行结束符处理。
```

File: migrate-from-csv-files-to-tidb.md [hunk 3]
```diff
@@ -61,6 +61,8 @@ data-source-dir = "${data-path}" # 本地或 S3 路径，例如：'s3://my-bucke
 separator = ','
 # 引用定界符，可以为零或多个字符。
 delimiter = '"'
+# 行结束符。默认将 \r、 \n、\r\n 都作为行结束符处理。
```
Comment (qiancai) [inline migrate-from-csv-files-to-tidb.md:64]:
```suggestion
# 行结束符。默认将 \r、\n、\r\n 都作为行结束符处理。
```

**Review by D3Hunter — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**